app/domain/authentication/authn_jwt/fetch_jwt_signing_key_factory.rb

```diff
@@ -0,0 +1,44 @@
+module Authentication
+  module AuthnJwt
+    # Factory that returns the interface implementation of FetchSigningKey
+    class FetchJwtSigningKeyFactory
+
+      def initialize(authenticator_parameters)
+        @authenticator_input = authenticator_parameters
+        @fetch_provider_uri_signing_key = Authentication::AuthnJwt::FetchProviderUriSigningKey.new(@authenticator_input)
+        @fetch_jwks_uri_signing_key = Authentication::AuthnJwt::FetchJwksUriSigningKey.new(@authenticator_input)
```

Comment:
i don't think that we need these. We can just use the actual objects in the `create` methods.